load-tests/src/fastauth/mod.rs

```diff
@@ -29,14 +30,35 @@ use near_primitives::{
     types::AccountId,
 };
 use primitives::UserSession;
-use rand::{distributions::Alphanumeric, Rng};
 use utils::build_send_and_check_request;
 
 pub async fn prepare_user_credentials(user: &mut GooseUser) -> TransactionResult {
     tracing::info!("prepare_user_credentials");
-    // Generate 2 key pairs
-    let fa_sk = SecretKey::from_random(near_crypto::KeyType::ED25519);
-    let la_sk = SecretKey::from_random(near_crypto::KeyType::ED25519);
+
+    let worker = near_workspaces::testnet().await.unwrap();
+
+    let root_account = Account::from_secret_key(
+        near_workspaces::types::AccountId::try_from("dev-1660670387515-45063246810397".to_string()).unwrap(),
+        near_workspaces::types::SecretKey::from_str(
+            "ed25519:4hc3qA3nTE8M63DB8jEZx9ZbHVUPdkMjUAoa11m4xtET7F6w4bk51TwQ3RzEcFhBtXvF6NYzFdiJduaGdJUvynAi"
+        ).unwrap(),
+        &worker
+    );
```

Comment:
the issue with this is that somebody can easily disrupt the load test by getting this key. We wouldn't notice this either until we run it locally manually. We should be creating a dev account each time and delete the account at the end of the test

Reply:
Yep, I know. But `dev_create_account` does not work for me. And there is [no Error handling](https://github.com/near/near-workspaces-rs/issues/353).